builtin/gc.c

```diff
@@ -1504,6 +1637,8 @@ static void initialize_maintenance_strategy(void)
 		tasks[TASK_LOOSE_OBJECTS].schedule = SCHEDULE_DAILY;
 		tasks[TASK_PACK_REFS].enabled = 1;
 		tasks[TASK_PACK_REFS].schedule = SCHEDULE_WEEKLY;
+		tasks[TASK_VFS_CACHE_MOVE].enabled = 1;
```

Comment:
This is modifying the `incremental` strategy for everyone, but this task is a no-op. That should be fine.

In older versions, `scalar` would configure the schedule manually, but [that appears to no longer be the case](https://github.com/microsoft/git/blob/1d7817ccfe14b7c0b9e2f9831cc1b6f0214b663f/scalar.c#L161) and instead we rely on [the default behavior of `git maintenance start`](https://github.com/microsoft/git/blob/1d7817ccfe14b7c0b9e2f9831cc1b6f0214b663f/scalar.c#L254-L265). So this is the cleanest way to upgrade folks with Scalar enlistments.

You may also want to add this task [to `scalar run`](https://github.com/microsoft/git/blob/1d7817ccfe14b7c0b9e2f9831cc1b6f0214b663f/scalar.c#L1160-L1166).

Reply:
> You may also want to add this task [to scalar run](https://github.com/microsoft/git/blob/1d7817ccfe14b7c0b9e2f9831cc1b6f0214b663f/scalar.c#L1160-L1166).

Not sure if this is required for newer tasks. The task mapping in: https://github.com/microsoft/git/blob/1d7817ccfe14b7c0b9e2f9831cc1b6f0214b663f/scalar.c#L1160-L1166
..feels primarily for backwards compatibility, for users/scripts using the `scalar` command to run those tasks that existed before with different names, rather than their `git maintenance`. names/command.

Thoughts @dscho?

Reply:
Sometimes I have recommended users run `scalar run` instead of `git maintenance run --schedule=weekly` for simplicity. But I could change that habit.

Reply:
I do like the option of asking users to call `scalar run` to clean up things like the `.pack` files that should go to the shared object directory.